Service/FilterService.php

```diff
@@ -128,6 +161,16 @@ public function getUrlOfFilteredImageWithRuntimeFilters($path, $filter, array $r
             $resolver
         );
 
+        if ($this->webpGenerate) {
+            // add webp in new location
+            $filteredWebpBinary = $this->createFilteredWebpBinary($path, $filter);
```

Comment:
I suggest add here new variable, like `$runtimeWebpPath = $runtimePath.'.webp'` and in 2 lines under use this variable instead doing two times concatenation.